### ISSUE TITLE
Network plugin does not work on the iOS (react-native 0.72.3)

### ISSUE THREAD

**Issue by shockwave13** [facebook/flipper#4969]
Network plugin does not work on the iOS  (working fine on android).

The application connects successfully, I can see the console logs.
But only these 4 requests are output to the network and that's it
<img width="707" alt="image" src="https://github.com/facebook/flipper/assets/11630403/f770a936-60b5-49a9-8a52-5639f5a99e17">

**react-native version is:** "react-native": "0.72.3",

I tried different versions of flipper. For example 0.203.0 and 0.176.1

Configure as described [here](https://fbflipper.com/docs/getting-started/react-native/). Tried both automatic and manual setup but none of the options helped. 

Setup doctor say that there are no problems with the configuration.
<img width="476" alt="image" src="https://github.com/facebook/flipper/assets/11630403/5d238655-26ba-4970-b37c-a404eab584b0">

Will be appreciated for help!

**Comment by alvaroraminelli-cb** [comment]
+1, facing the same issue after running `pod install`

```
[!] CocoaPods could not find compatible versions for pod "SocketRocket":
  In Podfile:
    FlipperKit/SKIOSNetworkPlugin (= 0.211.0) was resolved to 0.211.0, which depends on
      FlipperKit/FlipperKitNetworkPlugin (= 0.211.0) was resolved to 0.211.0, which depends on
        FlipperKit/Core (= 0.211.0) was resolved to 0.211.0, which depends on
          SocketRocket (~> 0.7.0)

    React-Core/RCTWebSocket (from `../../../../node_modules/react-native/`) was resolved to 0.72.3, which depends on
      SocketRocket (= 0.6.1)
```

**Comment by stefan-schweiger** [comment]
As far as I can see the last version which uses SocketRocket 0.6.1 is Flipper 0.203.0

So when I use this version the build works without a problem, but the Network plugin still does not work.

**Comment by amonraRivers** [comment]
same issue here

**Comment by alzalabany** [comment]
any updates ? :/

**Comment by raybelisle** [comment]
Yeah, same issue here. Would be good to have someone from Flipper comment?

**Comment by serhiy-z-israelit-pro** [comment]
> try deleting Flipper-related inits from `AppDelegate.m` file. That what helped after bumping 1 major to 0.72.x and having similar networking issues as you guys have described

this does fix the issue when your project uses `@react-navigation/native`, but if you are using `react-native-navigation` (by wix, v7.35.0) flipper won't be able to connect to ios app at all, not sure if it's fixed in latest releases of `react-native-navigation`

**Comment by HarshitMadhav** [comment]
I was facing the same issue so in `AppDelegate.m` file I just commented out the network plugin initialisation and restarted the app, network plugin then started working fine.

**Comment by raybelisle** [comment]
> I was facing the same issue so in `AppDelegate.m` file I just commented out the network plugin initialisation and restarted the app, network plugin then started working fine.

Could you show what you changed? I thought I removed the values but no change. Not sure if I'm not referencing the correct item?

**Comment by HarshitMadhav** [comment]
@raybelisle edited my comment

**Comment by sato-ryoka** [comment]
I created a new Expo project earlier and tried the following, but I got the same error.

I tried it with the stable react-native versions (0.74.2, 0.73.8, 0.71.19, 0.70.15), but it didn't work.
Regarding 0.70.15, it was not possible because the version did not match with React.

```
npx expo prebuild
cd ios
pod install
```

**Comment by sato-ryoka** [comment]
I don't know what the direct cause is, but after reinstalling cocoapods cli, I moved to the `ios` directory and did a `pod update`, and the error disappeared. There was no error during prebuild.
I hope it helps someone. ^^;
